src/week1/day0/gallery.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Gallery</title>
+    <link rel="stylesheet" href="./css/normalize.css">
+    <link rel="stylesheet" href="./css/gallery.css">
+</head>
+
+<body>
+    <section class="container">
+        <h1>Laptops</h1>
+        <div class="wrapper">
+            <div class="box">
+                <img src="./images/asus.jpg" alt="Asus Laptop Image">
+                <p class="title">
+                    Asus <br>
```

Comment:
You should not use **<br>** for layout reasons - use display: block elements instead.

E.g: <span> Asus </span>